modules/multisignatures.js

```diff
@@ -90,7 +90,7 @@ Multisignatures.prototype.processSignature = function (transaction, cb) {
 					multisignatureTransaction.signatures.push(transaction.signature);
 					multisignatureTransaction.ready = Multisignature.prototype.ready(multisignatureTransaction, sender);
 
-					library.bus.message('signature', {transaction: transaction.transaction, signature: transaction.signature}, true);
+					library.bus.message('signature', { transaction: transaction.transactionId, signature: transaction.signature}, true);
```

Comment:
We should use same attribute here as well to to publish event, so the event receiver should validate it right against schema. 

```
library.bus.message('signature', { transactionId: transaction.transactionId, signature: transaction.signature}, true);
```

Or its more appropriate to just publish the transaction itself. So in future if we add more attributes to that transaction, that automatically been published. 

```
library.bus.message('signature', transaction, true);
```